src/sections/Newcomers/NewcomersPageWrapper.style.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+
+const NewcomersPageWrapper = styled.section`
+	max-width: 1170px;
+	display: block;
+	height: auto;
+	margin: 200px auto;
+
+	a {
```

Comment:
Remove this. This will be done centrally after we decide on the colors for links

Reply:
Yeah, Sure I Will do. And one more thing. In the [layer5.io](https://layer5.io/community/newcomers) site we don't have the video I added it in this. Should I have to keep that or not?

Reply:
You can remove it. Anyways it is not something we want to keep on the site